assets/prefabs/structures/dungeon/dungeon1x2StoneDoorFrame/dungeonSmallStorageRoom.prefab

```diff
@@ -30,8 +30,8 @@
             {
                 "position": [-1, 1, 5],
                 "items": [
-                    {"slot": 5, "blockFamiliy": "Core:CobbleStone", "amount": 50 },
-                    {"slot": 12, "itemPrefab": "core:pickaxe"}
+                    {"slot": 5, "blockFamiliy": "CoreBlocks:CobbleStone", "amount": 50 },
+                    {"slot": 12, "itemPrefab": "CoreBlocks:pickaxe"}
```

Comment:
The `pickaxe` is part of `CoreBlocks`?

Reply:
Ooh, nope, it is certainly not, that stayed in Core. Would appear to be an overzealous search and replace